### PR TITLE
5164: add other statistics endpoint

### PR DESCRIPTION
**Pull request by rachaelparris**

### CODE REVIEW

File: shared/src/business/useCases/caseStatistics/updateOtherStatisticsInteractor.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+const { Case } = require('../../entities/cases/Case');
+const { UnauthorizedError } = require('../../../errors/errors');
+
+/**
+ * updateOtherStatisticsInteractor
+ *
+ * @param {object} providers the providers object
+ * @param {object} providers.applicationContext the application context
+ * @returns {Promise} the promise of the createUser call
+ */
```
Comment (rachelschneiderman) [inline shared/src/business/useCases/caseStatistics/updateOtherStatisticsInteractor.js:17]: Nit: docs

File: shared/src/proxies/caseStatistics/updateOtherStatisticsProxy.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+/**
+ * updateOtherStatisticsInteractor
+ *
+ * @param {object} providers the providers object
+ * @param {object} providers.applicationContext the application context
+ * @param {string} providers.caseId the case id to add notes to
+ * @param {string} providers.caseNote the notes to add
+ * @returns {Promise<*>} the promise of the api call
+ */
```
Comment (rachelschneiderman) [inline shared/src/proxies/caseStatistics/updateOtherStatisticsProxy.js:12]: Nit: docs

**Review by rachelschneiderman — APPROVED** [review]

File: shared/src/business/useCases/caseStatistics/updateOtherStatisticsInteractor.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+  if (!isAuthorized(user, ROLE_PERMISSIONS.ADD_EDIT_STATISTICS)) {
+    throw new UnauthorizedError('Unauthorized for editing statistics');
+  }
+
+  const oldCase = await applicationContext
+    .getPersistenceGateway()
+    .getCaseByCaseId({ applicationContext, caseId });
+
+  const newCase = new Case(oldCase, { applicationContext });
```
Comment (sutt0n) [inline shared/src/business/useCases/caseStatistics/updateOtherStatisticsInteractor.js]: i'm probably being nitpicky, but what about `const newCase = new Case({ ...oldCase, damages, litigationCosts }, { applicationContext });` here to consolidate lines 33-34?

File: web-client/src/views/CaseDetail/Statistics.jsx [hunk 4]
```diff
@@ -30,7 +30,10 @@ export const Statistics = connect(
             )}
           </div>
         )}
-        {statisticsHelper.formattedStatistics ? (
+        {statisticsHelper.showNoStatistics && (
```
Comment (sutt0n) [inline web-client/src/views/CaseDetail/Statistics.jsx:33]: 👍

**Review by sutt0n — APPROVED** [review]

File: shared/src/business/useCases/caseStatistics/updateOtherStatisticsInteractor.js (hunk 3 above)
Comment (rachaelparris) [reply shared/src/business/useCases/caseStatistics/updateOtherStatisticsInteractor.js]: Updated! Thank you!

**Pull request merged**